### PR TITLE
Fixed routing in single ui application. Added header into main app co…

### PR DESCRIPTION
**Pull request by Ravennod**
…mponent.

### CODE REVIEW

**Comment by Ravennod** [comment]
In order to make routing work please install url rewrite tool for your IIS.
https://www.iis.net/downloads/microsoft/url-rewrite

**Comment by Ravennod** [comment]
admin_route
main_route

**Comment by Shooter75** [comment]
image
Але я ще не ставив `URL Rewrite`.

File: MedicalDocManagment.WebUI/app/services/authentication.service.ts [hunk 1]
```diff
@@ -58,6 +58,14 @@ export class AuthenticationService {
     private grant_type:string = "password"
     constructor(private http: Http) {
         // set token if saved in local storage
+        //var currentUser = JSON.parse(localStorage.getItem('currentUser'));
```
Comment (KostetskyiYaroslav) [inline MedicalDocManagment.WebUI/app/services/authentication.service.ts:61]: If this commented lines are unnecessary remove it.

File: MedicalDocManagment.WebUI/app/admin/home/home.component.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<button class="btn btn-info" [routerLink]="['/add']">Додати користувача</button>
```
Comment (KostetskyiYaroslav) [inline MedicalDocManagment.WebUI/app/admin/home/home.component.html]: Don't forget to move this view into a suitable folder.

**Review by KostetskyiYaroslav — APPROVED** [review]

**Pull request merged**